services/sqs/src/main/java/software/amazon/awssdk/services/sqs/batchmanager/BatchOverrideConfiguration.java

```diff
@@ -35,13 +37,31 @@ public final class BatchOverrideConfiguration implements ToCopyableBuilder<Batch
     private final Integer maxBatchKeys;
     private final Integer maxBufferSize;
     private final Duration maxBatchOpenInMs;
+    private final Duration visibilityTimeout;
+    private final Integer longPollWaitTimeoutSeconds;
+    private final Duration minReceiveWaitTimeMs;
+    private final Integer maxDoneReceiveBatches;
+    private final List<String> receiveAttributeNames;
+    private final List<String> receiveMessageAttributeNames;
+    private final Boolean adaptivePrefetching;
+    private final Integer maxInflightReceiveBatches;
+    private final Boolean longPoll;
 
     public BatchOverrideConfiguration(Builder builder) {
-        //TODO : Add defaults based on QueueBufferConfig.java of V1 Default values.
         this.maxBatchItems = Validate.isPositiveOrNull(builder.maxBatchItems, "maxBatchItems");
-        this.maxBatchOpenInMs = Validate.isPositiveOrNull(builder.maxBatchOpenInMs, "maxBachOpenInMs");
         this.maxBatchKeys = Validate.isPositiveOrNull(builder.maxBatchKeys, "maxBatchKeys");
         this.maxBufferSize = Validate.isPositiveOrNull(builder.maxBufferSize, "maxBufferSize");
+        this.maxBatchOpenInMs = Validate.isPositiveOrNull(builder.maxBatchOpenInMs, "maxBatchOpenInMs");
```

Comment:
I was thinking the same, but I believe it would add unnecessary nesting. Since users are accustomed to V1, keeping the configuration the same will simplify migration. As this feature is primarily for parity, maintaining the V1 configuration will benefit customers transitioning to the new version.
Since these levels determine the buffer settings, the configuration should be set during the creation of the batch manager, not at the request level.